src/test/java/net/minecraftforge/debug/entity/living/LivingSetAttackTargetEventTest.java

```diff
@@ -0,0 +1,44 @@
+package net.minecraftforge.debug.entity.living;
```

Comment:
Missing license header.

Reply:
Fixed by c16f83fb8d8ee9619ea0d08bb7743d951591219e.

Reply:
Scratch that. My brain was working on a backwards order, thought this was 1.19.